sagemaker-triton/fil_ensemble/2_triton_xgb_fil_ensemble.ipynb

```diff
@@ -0,0 +1,908 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    while status == "Creating":
</pre><p>Consider using a Waiter rather than writing your own while-loop.</p><p><a href="https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/sagemaker.html#SageMaker.Waiter.EndpointInService" target="_blank">https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/sagemaker.html#SageMaker.Waiter.EndpointInService</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3524/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='51'/>

Reply:
Made appropriate code changes to call waiter